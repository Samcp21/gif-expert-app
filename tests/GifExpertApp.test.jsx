import { render, renderHook, screen } from "@testing-library/react";
import { GifExpertApp } from "../src/GifExpertApp";

describe("Pruebas en <GitExpertApp/>", () => {
  test("Match con el Snapshot", () => {
    // Arrange
    // Act
    const { container } = render(<GifExpertApp />);
    expect(container).toMatchSnapshot();
    screen.debug();
    // Assert
  });
  test("Valores iniciales", () => {
    // Arrange

    // Act
    render(<GifExpertApp />);
    const titulo = screen.getByRole("heading");
    console.log(titulo);
    // Assert
  });
});
