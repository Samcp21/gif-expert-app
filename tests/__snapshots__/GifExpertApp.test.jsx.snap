// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GitExpertApp/> Match con el Snapshot 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="Buscar Gif's"
      type="text"
      value=""
    />
  </form>
  <h3>
    One Punch
  </h3>
  <h2>
    Cargando...
  </h2>
  <div
    class="card-grid"
  />
  <h3>
    Dragon Ball
  </h3>
  <h2>
    Cargando...
  </h2>
  <div
    class="card-grid"
  />
</div>
`;
